base/pkg/entry.jl

```diff
@@ -155,31 +156,53 @@ end
 
 status(io::IO, pkg::AbstractString) = status(io, pkgname = pkg)
 
-function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool)
+function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool, avail::Dict, instd::Dict)
```

Comment:
Do any packages call this function? (How meta!) Would the changed number of arguments maybe break them?

Reply:
Nope. This is internal function which does status output for one packages. It should only be called from other wrapper functions.